src/main/java/com/havana/games/triovision/model/Board.java

```diff
@@ -6,10 +6,12 @@
 
 public class Board {
 
+	private static final int boardSize = 4; //Based on the fact that the board is square!
```

Comment:
static final variables are constants. Constants should follow the naming convention of UPPER_SNAKE_CASE.